README.md

```diff
@@ -139,12 +139,17 @@ You can deploy this project using:
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
+👤 **Sharon Kessio**
 
 - GitHub: [@kessio](https://github.com/kessio)
 - Twitter: [@kessio_sharon](https://twitter.com/kessio_sharon)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/sharon-kessio-172220b5)
 
+👤 **Shuhad Loofer**
+
+- GitHub: [@Shuhad786](https://github.com/Shuhad786)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/shuhad-loofer)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
```

Comment:
- [ ] this project should be invalidated by the first reviewer cause the readme file is not professional you shouldn't leave the support and acknowledgments sections empty with placeholders please make sure to add you fill them out with your information
<img width="894" alt="Screenshot 2023-02-09 at 14 33 18" src="https://user-images.githubusercontent.com/61790587/218129082-b5499dd1-0d6f-4d22-b5bf-7a4e1ef6a4d5.png">